var/spack/repos/builtin/packages/hugo/package.py

```diff
@@ -34,11 +35,12 @@ class Hugo(GoPackage):
     version("0.107.0", sha256="31d959a3c1633087d338147782d03bdef65323b67ff3efcec7b40241413e270a")
     version("0.106.0", sha256="9219434beb51466487b9f8518edcbc671027c1998e5a5820d76d517e1dfbd96a")
 
-    depends_on("go@1.11:", type="build", when="@0.48:")
-    depends_on("go@1.18:", type="build", when="@0.106:")
-    depends_on("go@1.20:", type="build", when="@0.123:")
-    depends_on("go@1.21.8:", type="build", when="@0.131:")
+    depends_on("go@1.23:", type="build", when="@0.144:")
```

Comment:
Based on the [0.144 release notes](https://github.com/gohugoio/hugo/releases/tag/v0.144.0), shouldn't this be `go@1.24:`?

Reply:
I'm guessing that may have been a typo on their part. They bumped it to `go@1.23:` in the file [here](https://github.com/gohugoio/hugo/commit/fd8b0fbf8a4efbe20c6a5143d5a9f308f08cab7a#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R173).

Reply:
Ok, weird. Looks like they continue to test that version.